scripts/copy_tob_wgs_mt.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+# Copy a subset of the data to the `test` bucket.
```

Comment:
This is not really a subset of the data, but a full copy.

I thought @vladsaveliev's pipeline can generate a true subset and write it to `test`?